study.md

```diff
@@ -147,5 +157,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+"My name is " + hank.given_name + " " + hank.family_name + " and I'm a " + hank.occupation + " that enjoys  " + hank.hobbies[0] + "."
```

Comment:
This will output the correct string, but we want to use template literals rather than string concatenation.

```javascript
const string = `My name is ${hank['given-name']} ${hank.family_name} and I'm a ${hank.occupation} that enjoys ${hank.hobbies[0]}.`;
```